mllm/models/deepseek_ocr/tokenization_deepseek_ocr.hpp

```
@@ -0,0 +1,51 @@
+#include <unordered_map>
+
+#include "mllm/models/ARGeneration.hpp"
+#include "mllm/preprocessor/tokenizers/BPE.hpp"
+#include "mllm/preprocessor/tokenizers/Unicode.hpp"
+#include "mllm/preprocessor/tokenizers/AutoTokenizer.hpp"
+
+namespace mllm::models::deepseek_ocr {
+
+// Actually is LlamaTokenizer
+class DpskOcrTokenizer final : public mllm::preprocessor::AutoTokenizer {
+  explicit DpskOcrTokenizer(const std::string& file_path) { preprocessor::initLocal(); }
```

Clang (14.0.6)

[error] constructor does not initialize these fields: bpe_

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)